lua/neo-tree/sources/buffers/lib/items.lua

```diff
@@ -72,7 +72,7 @@ M.get_opened_buffers = function(state)
       if #state.path > 1 then
         local rootsub = path:sub(1, #state.path)
         -- make sure this is within the root path
-        if rootsub == state.path then
+        if utils.normalize_path(rootsub) == utils.normalize_path(state.path) then
```

Comment:
`utils` has a method for this already which also does the normalization automatically:

```suggestion
        if utils.is_subpath(state.path, path) then
```

(also we can discard the rootsub local with this change)

Reply:
Thank you @pynappo, I've made another commit with these changes